ELJSBridge/Source/Global.swift

```diff
@@ -34,67 +34,67 @@ public protocol GlobalSupportable: JSExport {
      - parameter interval: The interval, in milliseconds, after which to call function.
      - returns: An identifier that can be used to cancel the call back.
      */
-    func set(fn: JSValue, interval: JSValue) -> UInt
+    func set(_ fn: JSValue, interval: JSValue) -> UInt
     
     /**
      Cancel a function scheduled to be called regularly with `setInterval`.
      
      - parameter identifier: The identifier returned by `setInterval`.
      */
-    func clearInterval(identifier: UInt)
+    func clearInterval(_ identifier: UInt)
 }
 
 /**
  This class implements functions commonly expected in the JS runtime like in a browser but are not present by default in the Bridge.
  For list of functions, see `GlobalSupportable`.
 */
 @objc
-public class Global: NSObject, GlobalSupportable, Scriptable {
+open class Global: NSObject, GlobalSupportable, Scriptable {
     
     // Stores all functions set up with setTimeout and setInterval along with their parameters. Removed after dispatch (in case of setTimeout) or when cleared.
     // Keeping all input values as JSValues so that they can be called on the JS function,  'fn',  in the dispatch and sent back as they came.
     var jsFunctions = [UInt: [JSValue]]()
     
-    public func set(fn: JSValue, timeout: JSValue) -> UInt {
+    open func set(_ fn: JSValue, timeout: JSValue) -> UInt {
         let identifier = UInt(arc4random())
         jsFunctions[identifier] = [fn, timeout]
         
-        dispatch_after(dispatch_time(DISPATCH_TIME_NOW, Int64(timeout.toInt32()) * Int64(NSEC_PER_MSEC)), dispatch_get_main_queue()) { () -> Void in
+        DispatchQueue.main.asyncAfter(deadline: DispatchTime.now() + Double(Int64(timeout.toInt32()) * Int64(NSEC_PER_MSEC)) / Double(NSEC_PER_SEC)) { () -> Void in
```

Comment:
This line of code seems a bit 🍌 to me. We got this fancy new `DispatchTime` in Swift 3, and the docs say
>DispatchTime represents a point in time relative to the default clock with nanosecond precision. On Apple platforms, the default clock is based on the Mach absolute time unit.

Is anyone more familiar with this API, What are the units supposed to be here? Maybe this is right if the value of `timeout` is in MSEC?

Let me know what you think. 

Not sure what the migrator was thinking. (The tests all still pass? 🤔 )